lambda/write-user-services/tests/write-user-services.test.ts

```diff
@@ -121,7 +112,7 @@ describe("validateUserServices", () => {
 
   describe("throws an error", () => {
     test("when user_id is missing", () => {
-      const userServices = parseRecordBody(
+      const userServices = JSON.parse(
```

Comment:
Can handle in a follow up, but did you see Paul's comment?

I think it was suggesting moving JSON.parsing to it's own function?

Can refactor later if that's what we want to do, will approve for now.

Reply:
I read his comment as the opposite - our parsing function was just calling `JSON.parse` so there was no need for it (which is why I got rid of it here)